validation/standard/versions.go

```diff
@@ -52,14 +53,38 @@ type VersionTags struct {
 
 var (
 	Release            Tag
-	NetworkVersions                           = make(map[string]VersionTags)
+	ContractVersions                          = make(map[string]map[Tag]superchain.ContractVersions)
 	BytecodeHashes     BytecodeHashTags       = make(BytecodeHashTags, 0)
 	BytecodeImmutables BytecodeImmutablesTags = make(BytecodeImmutablesTags, 0)
 )
 
 // L1ContractBytecodeHashes represents the hash of the contract bytecode (as a hex string) for each L1 contract
 type L1ContractBytecodeHashes superchain.ContractBytecodeHashes
 
+func (bch L1ContractBytecodeHashes) GetBytecodeHashFor(name string) (string, error) {
```

Comment:
Perhaps a unit test for this?

Reply:
Done in https://github.com/ethereum-optimism/superchain-registry/pull/768/commits/0647af459a5f5c57ae76575458cae111c644d4fd